operator/README.md

```diff
@@ -435,6 +425,8 @@ apiVersion: install.istio.io/v1alpha1
 kind: IstioOperator
 spec:
   components:
+    proxy:
+      enabled: false
```

Comment:
This is not right.
Component `proxy` has been removed.
```
out/linux_amd64/istioctl manifest generate -f operator/samples/pilot-advanced-override.yaml
Error: unknown field "proxy" in v1alpha1.IstioComponentSetSpec:
```

Reply:
This was taken from [samples/pilot-advanced-override.yaml](https://github.com/istio/istio/pull/22462/files#diff-61ea4e9ce6b970b83e8056b3a1fb37afR5). But if it is removed, I will update the actual sample yaml and corresponding README. 
Thanks